app/models/location.rb

```diff
@@ -0,0 +1,9 @@
+class Location < ApplicationRecord
```

Comment:
I was planning on geocoding by IP address so that the Frontend only needed to forward that to us. Do you think approaching it from an address is better? I'm not clear on where you are getting the address.